src/components/Modal/Modal.Body.tsx

```diff
@@ -97,10 +97,20 @@ export const Body = ({
         <div className={asideLabel} onClick={toggleAside}>
           {isAsideOpen && <>
             {labelClose}
-            <Icon color="currentColor" component={PiCaretRight} size={16} />
+            <Icon
+              aria-label={'PiCaretRight'}
```

Comment:
this is not a meaningful aria-label, also we are just circumventing the issue by hardcoding the previous labels